Acceptance-Tests.md

```diff
@@ -12,34 +12,46 @@ oppia/core/tests/
-1) The `spec` directory contains all the top-level test files. Each test file is named as `*.spec.js` and contains the test for a particular user type. For example, `blog-admin-tests` directory contains all the tests for the `Blog Admin` user.
+1) The `spec` directory contains all the top-level test files. Each test file is named as `*.spec.ts` and contains the test for a particular user type. For example, `blog-admin-tests` directory contains all the tests for the `Blog Admin` user.
 
 2) The `puppeteer-testing-utilities` directory contains all the utility files and helper functions, which you would require to write new acceptance tests. This directory can also be used to append more utility functions as when required or needed by the user.
 Files included inside this directory are :
-  - `puppeteer-utils.js` -> This file contains the base _*puppeteerUtilities*_ class which provides the most common and useful methods such as _*openBrowser*_, _*goto*_ etc. This class also serves the purpose of providing a base for defining other user oriented subclasses for eg. _*e2eBlogPostAdmin*_ class of _*blog-post-admin-utils.js*_ .
-  - `user-factory.js` -> This file contains methods for creating a certain user. The file has different methods for creating different types of user.
-  - `test-constants.js` -> This file contains defined constants such as _*URLs, classname, id etc. which are used in the tests.
-  - `show-message-utils.js` -> This file contains method for logging the progress and errors during a test.
+  - `puppeteer-utils.ts` -> This file contains the base _*puppeteerUtilities*_ class which provides the most common and useful methods such as _*openBrowser*_, _*goto*_, _*clickOn*_, _*openExternalPdfLink*_ etc. This class also serves as a foundation for defining other user-oriented subclasses, facilitating various testing scenarios.
+  - `user-factory.ts` -> This file contains methods for creating a certain user. The file has different methods for creating different types of user.
+  - `test-constants.ts` -> This file contains defined constants such as _*URLs, classname, id etc. which are used in the tests.
+  - `show-message-utils.ts` -> This file contains method for logging the progress and errors during a test.
 
-3) The `user-utilities` directory holds the utility files for different user types. Each user utility class is build upon the base `puppeteerUtilities` class containing the original methods along with the ones related to that user type. For eg. `e2eBlogPostAdmin` contains base functions as well as additional functions just related to `Blog Admin` user.
+3) The `user-utilities` directory holds the utility files for different user types. Each user utility class is build upon the base `puppeteerUtilities` class containing the original methods along with the ones related to that user type. For eg. `BlogPostEditor` contains base functions as well as additional functions just related to `Blog Admin` user.
```

Comment:
Change the puppeteerUtilities to BaseUser and also say that the BaseUser class is in the puppeteer-utils.ts

Reply:
3) The `user-utilities` directory holds the utility files for different user types. Each user utility class is built upon the base `BaseUser` class, which contains the original methods along with the ones related to that user type, and located in the `puppeteer-utils.ts` file. For example, `BlogPostEditor` contains both the base functions and additional functions specific to the `Blog Admin` user. 

@jnvtnguyen PTAL.

Reply:
Blog Admin -> Blog Post Editor. Other than that looks good.

Reply:
done